src/test/js/.eslintrc.yml

```diff
@@ -0,0 +1,16 @@
+# http://eslint.org/docs/rules/
+
+parserOptions:
+  ecmaFeatures:
+    impliedStrict: true
+    jsx: true
+  sourceType: module
+env:
+  browser: true
+  commonjs: true
+  jest: true
+  node: false
+rules:
+  import/namespace:
+    - 2
+    - allowComputed: true
```

Comment:
Is the point of this to disable rules from the top-level .eslintrc.yml, or what?

Reply:
Yes it overrides the rules from the top-level file. It came from the template project:

https://github.com/SFDO-Tooling/sfdo-template/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/test/js/.eslintrc.yml

The main purpose is presumably to make eslint "jest-aware". Without it you get:

   94:5  error  'expect' is not defined    no-undef

I haven't researched what each specific line does, but the file as a whole is needed.